src/Mod/PartDesign/Gui/Command.cpp

```diff
@@ -1122,6 +1122,20 @@ CmdPartDesignAdditiveLoft::CmdPartDesignAdditiveLoft()
 
 void CmdPartDesignAdditiveLoft::activated(int iMsg)
 {
+	App::Document *doc = getDocument();
+	PartDesign::Body *pcActiveBody = PartDesignGui::getBody(
+		/*messageIfNot = */ PartDesignGui::assureModernWorkflow(doc));
+	
+	// No PartDesign feature without Body past FreeCAD 0.13
+	if (!pcActiveBody) {
```

Comment:
Loft (as well as sweep and some others) are not available in the legacy part design, hence this workaround should not be used and the function should return directly.